libnetwork/drivers/bridge/setup_ip_tables.go

```diff
@@ -413,6 +429,11 @@ func setupInternalNetworkRules(bridgeIface string, addr *net.IPNet, icc, insert
 	if err := programChainRule(version, outDropRule, "DROP OUTGOING", insert); err != nil {
 		return err
 	}
+	if icc {
```

Comment:
Why add the bridge interface to firewalld only when ICC is enabled? For non-internal networks, the bridge interface is added to firewalld irrespective of the ICC setting. What's special about internal bridge networks with ICC disabled?